go/ct/gen/block_context.go

```diff
@@ -205,8 +209,34 @@ func (b *BlockContextGenerator) Generate(assignment Assignment, rnd *rand.Rand)
 	}
 
 	blobHashes := []vm.Hash{}
-	blobHashesCount := rnd.Intn(st.MaxDataSize)
-	for i := 0; i < blobHashesCount; i++ {
+	// we do not want this count to be zero, so we set it to a random value between 1 and MaxDataSize
+	blobHashesCount := rnd.Uint64n(st.MaxDataSize-1) + 1
```

Comment:
Heads-up: the number of generated blob hashes is part of the data produced by this generator. Depending on the set constraints, there may be a lower limit.

Also, the maximum should be single-digit, as the limit on Ethereum right now is 6.